### ISSUE TITLE
"Syntax Error" while running "Start Training"

### ISSUE THREAD

**Issue by HyzQuadriceps** [Linaqruf/kohya-trainer#103]
I used the scraper for around 300 images for this, I've already trained other loras with 15-20 images and I didn't have any problem

+-------------------------------+------------------------------------------------------------+
| Hyperparameter                | Value                                                      |
+-------------------------------+------------------------------------------------------------+
| v2                            | False                                                      |
| v_parameterization            | False                                                      |
| project_name                  | dd(giogio)style                                            |
| pretrained_model_name_or_path | /content/pretrained_model/AbyssOrangeMix2_nsfw.safetensors |
| vae                           | /content/vae/anime.vae.pt                                  |
| train_folder_directory        | /content/LoRA/train_data                                   |
| reg_folder_directory          | /content/LoRA/reg_data                                     |
| output_dir                    | /content/drive/MyDrive/LoRA/output                         |
| resume_path                   | False                                                      |
+-------------------------------+------------------------------------------------------------+
+--------------------------------+-------+
| Hyperparameter                 | Value |
+--------------------------------+-------+
| enable_bucket                  | True  |
| resolution                     | 512   |
| min_bucket_reso                | 256   |
| max_bucket_reso                | 1024  |
| bucket_reso_steps              | 64    |
| bucket_no_upscale              | False |
| cache_latents                  | True  |
| flip_aug                       | False |
| caption_extension              | .txt  |
| caption_dropout_rate           | 0     |
| caption_dropout_every_n_epochs | 0     |
+--------------------------------+-------+
+-------------------+---------------+
| Hyperparameter    | Value         |
+-------------------+---------------+
| network_dim       | 128           |
| network_alpha     | 128           |
| network_module    | networks.lora |
| network_weight    | False         |
| optimizer_type    | AdamW8bit     |
| optimizer_args    | False         |
| unet_lr           | 0.0001        |
| text_encoder_lr   | 5e-05         |
| lr_scheduler      | constant      |
| lr_warmup_steps   | 0             |
| lr_scheduler_args | 1             |
+-------------------+---------------+
+-----------------------------+------------------------------+
| Hyperparameter              | Value                        |
+-----------------------------+------------------------------+
| prior_loss_weight           | 1.0                          |
| lowram                      | True                         |
| noise_offset                | 0.1                          |
| max_train_type              | max_train_epochs             |
| max_train_type_value        | 20                           |
| train_batch_size            | 6                            |
| mixed_precision             | fp16                         |
| save_precision              | fp16                         |
| save_n_epochs_type          | save_n_epoch_ratio           |
| save_n_epochs_type_value    | 3                            |
| save_model_as               | safetensors                  |
| max_token_length            | 225                          |
| clip_skip                   | 2                            |
| gradient_checkpointing      | False                        |
| gradient_accumulation_steps | 1                            |
| seed                        | -1                           |
| logging_dir                 | /content/LoRA/logs           |
| additional_argument         | --shuffle_caption --xformers |
+-----------------------------+------------------------------+
./train.sh: line 2: syntax error near unexpected token `('
./train.sh: line 2: `accelerate launch --config_file=/content/kohya-trainer/accelerate_config/config.yaml --num_cpu_threads_per_process=8 train_network.py         --output_name=dd(giogio)style   --pretrained_model_name_or_path=/content/pretrained_model/AbyssOrangeMix2_nsfw.safetensors   --vae=/content/vae/anime.vae.pt   --train_data_dir=/content/LoRA/train_data   --reg_data_dir=/content/LoRA/reg_data   --output_dir=/content/drive/MyDrive/LoRA/output        --network_dim=128   --network_alpha=128   --network_module=networks.lora            --optimizer_type=AdamW8bit      --learning_rate=0.0001   --unet_lr=0.0001   --text_encoder_lr=5e-05   --lr_scheduler=constant              --enable_bucket   --resolution=512   --cache_latents   --bucket_reso_steps=64      --min_bucket_reso=256   --max_bucket_reso=1024   --caption_extension=.txt                 --prior_loss_weight=1.0   --lowram   --noise_offset=0.1   --train_batch_size=6   --max_train_epochs=20      --mixed_precision=fp16   --save_precision=fp16      --save_n_epoch_ratio=3   --save_model_as=safetensors   --max_token_length=225         --gradient_accumulation_steps=1   --clip_skip=2   --logging_dir=/content/LoRA/logs   --log_prefix=dd(giogio)style   --shuffle_caption --xformers'
